config/config.go

```diff
@@ -219,6 +219,9 @@ type Config struct {
 	// TLS version
 	TLSMinVersion string `yaml:"tls-min-version"`
 
+	// Exclude insecure cipher suites
+	ExcludeInsecureCipherSuites bool `yaml:"exclude-insecure-cipher-suites"`
```

Comment:
Do we want to exclude from a default set or rather enumerate allowed?

Reply:
It might be worthy of a discussion. I've updated the description with the options considered to allow for secure cipher suites.

I don't have a strong preference and went with the one requiring less maintenance.

Reply:
So if Go stdlib adds more "secure" ciphers than nobody needs to do anything and it will deployed automatically as a side effect.
This is good because likely Go stdlib will include only more secure cipher suites and not less secure ones. Of course these could become broken and you have to know the side effect and exclude these, too.
The bad thing is for people that have regulations like FIPS, which may or may not be up to date with secure cipher suite definitions.

Right now I have no opinion on which one is better, but maybe rather to say we should have an allow list and an exclude list for giving the user a choice based on their needs.

Reply:
I'd favor having an optional allow list for that case, since it's likely one would want to be specific about what's allowed.

Reply:
I think we should start with a single boolean flag to disable insecure suites and then add a list flag if we even find a need for it.